base_layer/wallet/src/transaction_service/service.rs

```diff
@@ -1842,12 +1843,20 @@ where
     pub async fn accept_recipient_reply(
         &mut self,
         source_pubkey: CommsPublicKey,
-        recipient_reply: proto::RecipientSignedMessage,
+        recipient_reply: Result<proto::RecipientSignedMessage, prost::DecodeError>,
     ) -> Result<(), TransactionServiceError> {
         // Check if a wallet recovery is in progress, if it is we will ignore this request
         self.check_recovery_status()?;
 
+        if let Err(e) = recipient_reply {
+            // TODO: We should ban but there is no banning in the wallet...
```

Comment:
```suggestion
            // We should ban but there is no banning in the wallet...
```
created an issue for this: https://github.com/tari-project/tari/issues/5694